tests/nimfs.rs

```diff
@@ -0,0 +1,106 @@
+#![cfg(feature = "hypernova")]
```

Comment:
Which would be the reason of having this file is in a newly created directory `tests/` instead of living with the rest of multifolding code in `src/ccs/`?

Reply:
That this forces us to just use the public exported API. Hence, we can see how the final API usage looks like. As well as visibility etc..